src/Microsoft.DotNet.GenAPI/Shared/CSharpSyntaxWriter.cs

```diff
@@ -131,9 +159,15 @@ public void WriteAttribute(string attribute)
     }
 
     /// <inheritdoc />
-    public void WriteProperty(string definition, bool hasImplementation, bool hasGetMethod, bool hasSetMethod)
+    public void WriteProperty(IEnumerable<SyntaxKind> accessibilities, string definition, bool hasImplementation, bool hasGetMethod, bool hasSetMethod)
```

Comment:
Imho this is better:
```suggestion
    public void WriteProperty(IEnumerable<SyntaxKind> accessibilityModifiers, string definition, bool hasImplementation, bool hasGetMethod, bool hasSetMethod)
```

applies below as well